docs/jazzy.yml

```diff
@@ -182,10 +182,10 @@ custom_categories:
       - RoadObjectMatcher
       - RoadObjectsStore
```

Comment:
#3045 renamed `RoadObjectsStore` and `RoadObjectsStoreDelegate` to `RoadObjectStore` and `RoadObjectStoreDelegate`, respectively. I guess those names should stay as is, because `Store` and `StoreDelegate` would be misleading in any code that relies on type inference. (It would only be slightly less weird if we called it `Storage`.) In any case, these entries in the table of contents need to be updated to match the declared type names.